types/euwa-wrapper/test/euwa-wrapper.tests.ts

```diff
@@ -0,0 +1,39 @@
+import EUWA from 'euwa-wrapper';
```

Comment:
this seems wrong, there is no 'euwa-wrapper' package in the ecosystem,
I could be wrong here, though. Is this real life-like code sample?

Reply:
Hi, I used their real life example found here under the heading Basic Usage: [click here](https://help.puzzel.com/product-documents/user-guide/puzzel-contact-centre/puzzel-administration-portal/services/chat-configuration/euwa-wrapper-interface)

I used this same type in our website sats.no. Puzzel did not have any types published so I thought to publish the one I made and used in our site.

thanks for reviewing, all tips welcome this is my first time :-)

Reply:
oh yeah their registry is not on npm :-( 

```
Connecting to the NPM Registry
A .npmrc file should be created either in your project or on user level. Read more about .npmrc on https://docs.npmjs.com/cli/v6/configuring-npm/npmrc

The following lines should be added:

@puzzel:registry=https://puzzel.pkgs.visualstudio.com/public/_packaging/main/npm/registry/
always-auth=true
```

Reply:
I've used their instruction to download their package, and it indeed doesn't have types. But, his package is not on `npm`, not a global script (like jquery, youtube, etc), and not a runtime (node, bun, etc).

Additionally, there doesn't seem to be a license on that page that says whether or not these types can be copied and republished. The package that gets downloaded doesn't include a license at all!

I'm not convinced that this should (or can) be published on DefinitelyTyped. Can the authors be convinced to publish their code on npm with the the types? Or at least publish in a public registry with some sort of license on the types?

Reply:
Thanks @jakebailey for double checking, I will contact them and ask them if they want to do this. In the meantime should I put this PR draft mode again while I discuss with them, or should I close the PR?

Reply:
Whichever; the bot will probably close this if it's not merged in a few weeks, so a draft may be more representative.